Hangman.c

```diff
@@ -0,0 +1,160 @@
+/*Lloyd Page*/
+/*Hang man game with ASCII art, rand word/userinputed word, repeatable*/
```

Comment:
Amazing job on this assignment, Lloyd! There was one warning returned and you should have included more comments, but great job nonetheless! 💯